bucket/diagram-designer.json

```diff
@@ -0,0 +1,18 @@
+{
+    "version": "1.29.5",
+    "description": "Vector graphics editor for creating flowcharts, UML class diagrams, illustrations and slide shows",
+    "homepage": "https://logicnet.dk/DiagramDesigner/",
+    "license": "Freeware",
+    "url": "https://www.fosshub.com/Diagram-Designer.html/DiagramDesignerSetup.1.29.5.msi",
```

Comment:
Is it 32 bit?

Reply:
It is a 32-bit application. However, if I put the url under *architecture -> 32bit*, it will cause an error when installing under 64bit systems. (`ERROR 'xxx' doesn't support 64bit architecture!`)

I personally think that we can discard 32-bit support, as there are few PCs running under 32-bit, but that requires further discussion with the maintainers.

```
(In VS2019 developer command prompt)
C:\> dumpbin /headers D:\scoop\apps\diagram-designer\current\DiagramDesigner.exe
Microsoft (R) COFF/PE Dumper Version 14.29.30133.0
Copyright (C) Microsoft Corporation.  All rights reserved.


Dump of file D:\scoop\apps\diagram-designer\current\DiagramDesigner.exe

PE signature found

File Type: EXECUTABLE IMAGE

FILE HEADER VALUES
             14C machine (x86)
               3 number of sections
        2A425E19 time date stamp Sat Jun 20 06:22:17 1992
               0 file pointer to symbol table
               0 number of symbols
              E0 size of optional header
            818F characteristics
                   Relocations stripped
                   Executable
                   Line numbers stripped
                   Symbols stripped
                   Bytes reversed
                   32 bit word machine
```

Reply:
No no

If its 32 bit, this scope is correct